code/software/chardevs/README.md

```diff
@@ -0,0 +1,38 @@
+# C Starter Project for rosco_m68k
+
+This is an empty (well, "Hello World") C project. You can use it as 
+a starting point for your own programs.
+ 
+## What next?
+
+* Edit `kmain.c` to add your own code.
+* Add more C or assembly files. Add them to the `$(OBJECTS)` line in `Makefile`
+  * Use the `.o` extension instead of `.c` or `.S` in the `Makefile`!
+* Make this README.md your own!
```

Comment:
This `chardevs` example still has the original C Starter Project README and Makefile header comment. To avoid confusion, it'd be good to update the beginning part to match the code.

Reply:
Good spot, will do, thanks :)